### PR TITLE
Add option notify_on

### PR DESCRIPTION
**Pull request by koudaiii**
## WHY

https://github.com/wantedly/step-pretty-slack-notify/issues/17

## WHAT

* Add Option: notify_on specific `failed` or `passed` only

### Usage 
```yml
build:
    after-steps:
        - wantedly/pretty-slack-notify:
            webhook_url: $SLACK_WEBHOOK_URL
            notify_on: "failed"
```

### CODE REVIEW

File: run.rb [hunk 1]
```diff
@@ -61,10 +61,12 @@ def username_with_status(username, status)
 
 notifier.channel = '#' + channel if channel
 
-res = notifier.ping(
-  message,
-  icon_url: icon_url(ENV["WERCKER_RESULT"])
-)
+if  notify_on.empty? || notify_on == ENV["WERCKER_RESULT"]
```
Comment (dtan4) [inline run.rb]: Extra space :sunglasses:

**Comment by koudaiii** [comment]
Thank you for review!
Delete extra space :bow:

File: README.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ Options
 * `channel`  - The Slack channel to override the default channel. (without #).
 * `username` - The name of your bot. (default `Wercker`)
 * `branches` - Specific branches to notify. (regular expression)
+* `notify_on` - Allows you to specify whether to notify on failures only
```
Comment (dtan4) [inline README.md]: I understand that `notify_on` is `passed` or `failed` only.
I think it is better to explain about valid values :bulb: 

e.g.

```
* `notify_on` - Allows you to specify whether to notify on `passed` or `failed`.

**Comment by koudaiii** [comment]
Thank you for review :)

How about this ? @dtan4 

e.g.
```
* `notify_on` - Allows you to specify to notify on `passed` or `failed`. (default all allows notify)
```

**Comment by dtan4** [comment]
Great :+1:

**Comment by dtan4** [comment]
LGTM :+1:

**Pull request merged**